scripts/validate-structured-data.py

```
@@ -0,0 +1,216 @@
+            errors.append("BreadcrumbList missing 'itemListElement'")
+    
+    elif schema_type == "JobPosting":
+        required = ["title", "description", "datePosted", "hiringOrganization"]
+        for prop in required:
+            if prop not in schema_data:
+                errors.append(f"JobPosting missing '{prop}'")
+    
+    return errors
+
+
+def validate_structured_data(json_ld: str, context: str = "") -> Dict[str, Any]:
```

Ruff (0.14.3)

Unused function argument: `context`

(ARG001)